i18n/en/docusaurus-plugin-content-docs/current/guides/tech/with-nuxtjs.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 10
+---
+# Use with NuxtJS
+
+It is possible to implement FSD in a NuxtJS project, but conflicts arise due to differences between the NuxtJS project structure requirements and FSD principles: 
+
+- Initially, NuxtJS offers a project file structure without a `src` folder, i.e. in the root of the project.
+- The file routing is in the `pages` folder, while in FSD this folder is reserved for the flat slice structure.
+
+
+## Adding an alias for the `rc` directory
+
+Add an `alias` object to your config:
+
+```ts
+export default defineNuxtConfig({
+  devtools: { enabled: true },      // Not related to FSD, enabled at project startup
+  alias: {
+    "@": '~/src'
+  },
+})
+```
+
+## Move file routing to `src/app`.
+
+First of all, create a `src` directory in the root of your project, and create app and pages layers inside this directory.
+Thus, your file structure should look like this:
+
+```sh
+├─── src
+│ ├─── app           
+│ ├─── pages                        # The pages folder assigned to FSD
+```
+
+In order for NuxtJS to use the `app` layer for file routing, you need to modify `nuxt.config.ts` as follows:
+```ts
+export default defineNuxtConfig({
+  devtools: { enabled: true },      // Not FSD related, enabled at project startup
+  alias: {
+    "@": '~/src'
+  },
+  dir: {
+    pages: './src/app'
+  }
+})
+```
+
+Now, you can create roots for pages within `app` and connect pages from `pages` to them. 
+
+For example, to add a `Home` page to your project, you need to do the following steps:
+- Add a page slice inside the `pages` layer
+- Add the corresponding root inside the `app` layer
+- Align the page from the slice with the root
+
+To create a page slice, let's use the [CLI](https://github.com/feature-sliced/cli):
+
+```shell
+  fsd p home -s ui
```

Comment:
suggestion: I prefer to write out full commands in documentation because it would help people understand at a glance what the command would do. Shorthands exist more for everyday interactive usage. Also, slices in the Pages layer get the UI segment automatically, so we can omit that parameter.
```suggestion
fsd pages home
```